homeassistant/components/zha/light.py

```diff
@@ -438,6 +441,28 @@ def async_restore_last_state(self, last_state):
         if "effect" in last_state.attributes:
             self._effect = last_state.attributes["effect"]
 
+    async def async_turn_on(self, **kwargs):
+        """Turn the entity on."""
+        await self._handle_on_off(super().async_turn_on, kwargs)
+
+    async def async_turn_off(self, **kwargs):
+        """Turn the entity off."""
+        await self._handle_on_off(super().async_turn_off, kwargs)
+
+    async def _handle_on_off(self, operation, kwargs):
+        """Perform the on or off operation."""
+        transition = kwargs.get(light.ATTR_TRANSITION)
+        default_transition = DEFAULT_TRANSITION / 10 if DEFAULT_TRANSITION > 1 else 1
```

Comment:
DEFAULT_TRANSITION is already a constant. Is this really needed?

Reply:
Yes they are in different scales. In the ZCL command it is 1/10th a second in the call later it is seconds.

Reply:
But why if/else if it is a constant? Unless it is a not constant constant

Reply:
Did you mean `transition` in the if/else instead of the default_transition

Reply:
Also, folks use the constant modifier to change the value of this... so this is here to handle that. I have another PR that will allow changing this from the UI as well.

Reply:
> Did you mean `transition` in the if/else instead of the default_transition

Nope. See above. Folks change this with constant modifier because we don’t currently support customization.

Reply:
```suggestion
        default_transition = max(DEFAULT_TRANSITION / 10, 1)
```